src/__tests__/fixtures/color-case.compressed.css

```diff
@@ -1 +1 @@
-p{color:#AA93FE;background-color:#AFE}
\ No newline at end of file
+p{color:#AA93FE;background:#FFF url(/ImagesDir/loader.gif) center center no-repeat}
```

Comment:
As I can see, the case from https://github.com/ben-eb/perfectionist/issues/43 is going to be fixed with this PR merge.